templates/harbor.yml.epp

```diff
@@ -120,6 +126,18 @@ chart:
 log:
   # options are debug, info, warning, error, fatal
   level: <%= $log_level %>
+
+<% if ersioncmp($cfg_version, '1.9.0') >= 0 { -%>
```

Comment:
missing v?

Reply:
is $cfg_version not the same as the on in config.pp

Reply:
I meant versioncmp is missing the leading character -- `ersioncmp`